sharepoint/sharepoint-ps/sharepoint-online/Start-SPOCopilotAgentInsightsReport.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spocopilotagentinsightsreport
+applicable: SharePoint Online
+title: Start-SPOCopilotAgentInsightsReport
+schema: 2.0.0
+author: bhagatshweta
+ms.author: bhagatshweta
+ms.reviewer:
+manager: hikakar
+---
+ 
+# Start-SPOCopilotAgentInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables administrator to trigger the build of a new Copilot Agent Insight report for the last N days.
+
+## SYNTAX
+
+```powershell
+Start-SPOCopilotAgentInsightsReport [-ReportPeriodInDays <Int>] [-Force <SwitchParameter>]
+```
+
+## DESCRIPTION
+
+After this cmdlet is executed, the Copilot Agent Insight report generation request for the last N days gets queued in the pipeline and the below metadata is displayed with the following properties:
+
+| Property             | Description                                                 |
+|:---------------------|:------------------------------------------------------------|
+| Id                   | The unique Id of the report.                                |
+| CreatedDateTimeInUtc | The date and time the report creation was triggered in UTC. |
```

Comment:
>Created [](http://example.com/codeflow?start=2&length=7)

this should be present, so "Create"

Reply:
hi @samkabue , it would be CreatedDateTimeInUtc according to the design doc: 
![image](https://github.com/user-attachments/assets/e36454b6-84a7-45ef-8c34-dd6c64b917b3)